release-notes/6.0/preview/api-diff/preview7/.Net/6.0.0-preview7_System.Diagnostics.md

```diff
@@ -0,0 +1,54 @@
+# System.Diagnostics
+
+``` diff
+ namespace System.Diagnostics {
+     public static class Debug {
++        public static void Assert(bool condition, Debug.AssertInterpolatedStringHandler message);
++        public static void Assert(bool condition, Debug.AssertInterpolatedStringHandler message, Debug.AssertInterpolatedStringHandler detailMessage);
++        public static void WriteIf(bool condition, Debug.WriteIfInterpolatedStringHandler message);
++        public static void WriteIf(bool condition, Debug.WriteIfInterpolatedStringHandler message, string category);
++        public static void WriteLineIf(bool condition, Debug.WriteIfInterpolatedStringHandler message);
++        public static void WriteLineIf(bool condition, Debug.WriteIfInterpolatedStringHandler message, string category);
++        public struct AssertInterpolatedStringHandler {
```

Comment:
Some of these types are attributed as EditorBrowsableState.Never because they're not supposed to be used directly.  Do we want to call them out in the diff?  Or should we ensure that [EditorBrowsable] attributes are present in these files?

Reply:
the api-reviewer doesnt work on any kind of attributes, it would be not scalable to add them manually, we can take a look at the tool to improve this.